### DISCUSSION TITLE
Example: Epic Stack + Stripe

### DISCUSSION THREAD

**Discussion by kentcdodds** [epicweb-dev/epic-stack#684] [Announcements]
I'm thrilled to share that @saurabhp75 has put together an example of the Epic Stack with Stripe! It's full featured. If you're looking to build a SaaS app or something I think you'll really love this!

Give it a look and a star here: https://github.com/saurabhp75/epic-stripe

Check out other examples here: https://github.com/epicweb-dev/epic-stack/blob/main/docs/examples.md

**Comment by herringtown** [comment]
@saurabhp75 nice job & thank you!

**Reply by saurabhp75** [reply]
Glad my work made a positive impact. Thank you!
